src/Spinner/Spinner.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { themeGet } from '@styled-system/theme-get';
+import { withTheme } from 'emotion-theming';
+import Loader from 'react-loader';
+
+const Spinner = ({ children, color, spin, theme, ...props }) => {
+  const themedColor = themeGet(`colors.${color}`, color)({ theme });
```

Comment:
Since the theme is only being destructured above for the use in `themeGet` could you just pass props in whole sale? Then you could avoid the prop-type declarations as well since `withTheme` will handle having theme.

I mostly ask because the example usage in the docs show just passing props in wholesale. https://www.npmjs.com/package/@styled-system/theme-get

Reply:
Good call 👍